ui/src/app/utils/user-apps-utils.tsx

```diff
@@ -33,6 +34,12 @@ export const appTypeToString: Record<AppType, string> = {
   [AppType.SAS]: 'SAS',
 };
 
+export const appMinDiskSize: Record<AppType, number> = {
```

Comment:
I like this.

In some cases, this differs from the `default_xxx_CreateRequest` values.  Should we change the default create quests to match these?  hmm.

Reply:
I made an attempt at this and it got more involved than I expected!  So I think we should leave this for now.